index.html

```diff
@@ -10178,16 +10178,72 @@ <h3>Value</h3>
 				<dt id="valuetype_string">string</dt>
 				<dd>Unconstrained value type.</dd>
 				<dt id="valuetype_token">token</dt>
-				<dd>One of a limited set of allowed values. An explicit value of <code>undefined</code> for this type is the equivalent of providing no value.</dd>
+				<dd>One of a limited set of allowed enumerated values. The default value is defined in each attribute's Values table, as specified in the <a href="#enumerated-attribute-values">Enumerated Attribute Values</a> section.</dd>
 				<dt id="valuetype_token_list">token list</dt>
 				<dd>A list of one or more tokens.</dd>
 			</dl>
 			<p>These are generic types for states and properties, but do not define specific representation. See <a href="#state_property_processing">State and Property Attribute Processing</a> for details on how these values are expressed and handled in host languages.</p>
 		</section>
 	</section>
-	<section id="state_prop_values">
-		<h2>Values for States and Properties</h2>
-		<p>Many <a>states</a> and <a>properties</a> accept a specific set of tokens as <span>values</span>. The allowed values and explanation of their meaning is shown after the table of characteristics. The default value, if defined, is shown in <strong>strong</strong> type, followed by the parenthetical term 'default'. When a value is indicated as the default, the user agent MUST follow the behavior prescribed by this value when the state or property is empty or unspecified. Some <a>roles</a> also define what behavior to use when certain states or properties, that do not have default values, are not provided.</p>
+	<section id="enumerated-attribute-values">
+		<h2>Enumerated Attribute Values</h2>
+		<p>When the ARIA attribute definition includes a table enumerating the attribute's allowed <span>values</span>, that attribute is an <a href="https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#enumerated-attribute">enumerated attribute</a>. When the values table notates one of the values as "(default)", that default value is the <a href="https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#missing-value-default">missing value default</a> and <a href="https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#invalid-value-default">invalid value default</a> for the attribute.</p>
```

Comment:
I suggest adding the sentence "Each value in the table is a keyword for the attribute, mapping to a state of the same name" after the first one.

Reply:
To make sure I understand you, by "after the first one" do you mean "after the first sentence"? Please confirm (👍 is fine) or clarify further. Thanks.

Reply:
> "Each value in the table is a keyword for the attribute, mapping to a state of the same name"

I'm having trouble with this sentence, since it uses the term "state" in a different way than how "states and properties" is used elsewhere in the spec. I tried to rid the ARIA spec of most instances of those terms at one point, but they've snuck back in. "state" is the right term here IMO, but perhaps there is another synonym that would avoid the ambiguity altogether.

Reply:
Perhaps 'condition'? "Each value in the table is a keyword for the attribute, mapping to a condition of the same name." I don't love it, but could live with it if no one objects.

Reply:
Enumerated attributes are defined by their states, so it doesn't work to use a different word. I recognize there's an unfortunate terminology collision here, but it's a term of art that goes along with enumerated attribute, so kind of unavoidable...